src/components/ProjectDashboard/hooks/useProjectHeader.ts

```diff
@@ -1,12 +1,42 @@
-// TODO: fill out data
-export const useProjectHeader = () => {
+import { V2V3ProjectContext } from 'contexts/v2v3/Project/V2V3ProjectContext'
+import { BigNumber } from 'ethers'
+import { useProjectTrendingPercentageIncrease } from 'hooks/useProjects'
+import { useContext } from 'react'
+import { useProjectMetadata } from './useProjectMetadata'
+
+export interface ProjectHeaderData {
+  title: string | undefined
+  subtitle: string | undefined
+  handle: string | undefined
+  projectId: number | undefined
+  owner: string | undefined
+  payments: number | undefined
+  totalVolume: BigNumber | undefined
```

Comment:
Avoid passing out BigNumber - I am trying to cut back on Big Number being exposed in our code where possible. It's mostly confusing and error prone

Reply:
lemme know what you think - happy to discuss this, but I feel pretty strongly that us passing BigNumber everywhere is an antipattern

Reply:
im down - what do you suggest instead?

Reply:
transforming data (kind of like a presenter) before passing back out to the component to be used. So for instance, totalVolume would be a string of something like "$1,000,000USD" or whatever we're doing, depending on currency

Reply:
@wraeth-eth I don't mind this but I'm also not entirely clear on the benefit for this case.

I want to continue using ETHAmount and USDAmount where possible, for maximum consistency throughout the app (has USD tooltip, formatted correctly etc).

```
    <ProjectHeaderStat
        label={t`Total volume`}
        stat={<ETHAmount amount={totalVolume} precision={2} />}
      />
      ```

I might make the argument for thinking of data in the opposite direction: keep the data unaltered from source (chain) until the component layer, which choses to transform the data based on its desired behavior.

Think im gonna keep as-is for now but keen to continue discussion - i do hate bignumber.